definitions/divorce/json/CaseField/CaseField-general-email-nonprod.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "LiveFrom": "08/09/2020",
```

Comment:
I think you forgot to add the authorisations for the fields?

Reply:
yes, you're right :)